packages/discord.js/src/structures/Guild.js

```diff
@@ -499,6 +499,13 @@ class Guild extends AnonymousGuild {
     } else {
       this.incidentsData ??= null;
     }
+
+    if (data.soundboard_sounds) {
+      this.soundboardSounds.cache.clear();
```

Comment:
If `soundboard_sounds` is only sent on `GUILD_CREATE`, then isn't this line useless? What will it clear apart from nothing?

Reply:
unavailable Guild becoming available again?

Reply:
It is not done for all similar properties.

I have to wonder why it's in `_patch()`. No gateway events will be received if the guild is unavailable nor can members do anything. Seems more sensible to be done in the constructor, no?

Reply:
Seems like `guild_scheduled_events`, `stage_instances`, `presences` and maybe more fields do have a similar check in `_patch()` while only being sent on `GUILD_CREATE` too.